### PR TITLE
Added automatic exporting AWS cloud to kentik in terraform module

### PR DESCRIPTION
**Pull request by lukaa12**
KNTK-212

### CODE REVIEW

File: cloud_AWS/terraform/module/variables.tf [hunk 1]
```diff
@@ -50,3 +50,41 @@ variable "create_role" {
...
 }
+
+variable "name" {
+  description = "Exported resource name in kentik portal"
+  type = string
+  default = "terraform_aws_exported_cloud"
+}
+
+variable "enabled" {
+  description = "Defines if cloud exported do kentik is enabled"
```
Comment (Fenthick) [inline cloud_AWS/terraform/module/variables.tf:61]:
```suggestion
  description = "Defines if cloud exported to Kentik is enabled"
```

File: cloud_AWS/terraform/module/variables.tf [hunk 2]
```diff
@@ -50,3 +50,41 @@ variable "create_role" {
   type        = bool
   default     = true
 }
+
+variable "name" {
+  description = "Exported resource name in kentik portal"
```
Comment (Fenthick) [inline cloud_AWS/terraform/module/variables.tf:55]:
```suggestion
  description = "Exported cloud name in Kentik Portal"
```

File: cloud_AWS/terraform/module/variables.tf [hunk 3]
```diff
@@ -50,3 +50,41 @@ variable "create_role" {
...
+}
+
+variable "enabled" {
+  description = "Defines if cloud exported do kentik is enabled"
+  type        = bool
+  default     = true
+}
+
+variable "description" {
+  description = "Description of exported resource in kentik portal"
```
Comment (Fenthick) [inline cloud_AWS/terraform/module/variables.tf:67]:
```suggestion
  description = "Description of exported resource in Kentik Portal"
```

File: cloud_AWS/terraform/module/variables.tf [hunk 4]
```diff
@@ -50,3 +50,41 @@ variable "create_role" {
...
+  default     = false
+}
+
+variable "multiple_buckets" {
+  type        = bool
+  default     = false
+}
+
+variable "region" {
+  description = "Specifies aws region passed to kentik-cloudexport provider"
```
Comment (Fenthick) [inline cloud_AWS/terraform/module/variables.tf:88]:
```suggestion
  description = "Specifies AWS region passed to kentik-cloudexport provider"
```

**Review by Fenthick — APPROVED** [review]
Only stylistic comments

File: .github/workflows/azure-ansible.yml [hunk 5]
```diff
@@ -19,11 +19,11 @@ jobs:
...
-      - name: Lint Role 
-        uses: github/super-linter@v3
-        env:
-          VALIDATE_ALL_CODEBASE: false
-          VALIDATE_ANSIBLE: true
-          ANSIBLE_DIRECTORY: /cloud_Azure/ansible/roles/kentik_az
-
-    
+      - name: Lint Role
+# TODO(lwolanin) reenable this before making any changes to azure ansible scripts
```
Comment (mateuszmidor) [inline .github/workflows/azure-ansible.yml:23]: maybe shortly explain why this is commented out

**Review by mateuszmidor — DISMISSED** [review]

File: .github/workflows/azure-ansible.yml (hunk 5 above)
Comment (mmac-m3a) [reply .github/workflows/azure-ansible.yml:23]: I generally prefer unused code/config to be removed as opposed to commented out, unless it contains information useful for understanding why the used code/config exists or where it is heading. In all other cases it just pollutes searches. This is case is as benign as it gets, so I'm fine with it.

**Review by mmac-m3a — APPROVED** [review]
LGTM

File: .github/workflows/azure-ansible.yml (hunk 5 above)
Comment (Fenthick) [reply .github/workflows/azure-ansible.yml:23]: @lukaa12 Please create a JIRA ticket to re-enable the linter

File: cloud_AWS/terraform/module/README.md [hunk 6]
```diff
@@ -59,6 +66,13 @@ module "kentik_aws_integration" {
 | iam\_role\_prefix | Prefix to use with IAM roles | `string` | `Kentik` | no |
 | store\_logs\_more\_frequently | Allows to chose how often save logs to s3. Default is once per 10 minutes. When enabled it saves once per minute | `bool` | `false` | no |
 | create\_role | If to create kentik role | `bool` | `true` | no |
+| name | Exported cloud name in Kentik Portal | `string` | `terraform_aws_exported_cloud` | no |
+| enabled | If cloud exported to Kentik is enabled | `bool` | `true` | no |
+| description | Description in Kentik Portal | `string` | `` | no |
+| plan\_id | Billing plan ID | `string` | | yes |
+| delete\_after\_read | If to delete after read | `bool` | `false` | no |
+| multiple\_buctets | If to use multiple buckets | `bool` | `false` | no |
```
Comment (Fenthick) [inline cloud_AWS/terraform/module/README.md:74]:
```suggestion
| multiple\_buckets | If to use multiple buckets | `bool` | `false` | no |
```

File: .github/workflows/azure-ansible.yml (hunk 5 above)
Comment (lukaa12) [reply .github/workflows/azure-ansible.yml:23]: DONE (https://clkentik.atlassian.net/browse/KNTK-220?atlOrigin=eyJpIjoiZDA3YmJiZmI4Y2E1NDE1Zjk4ZGE3NWY0YTk4MzNlNTkiLCJwIjoiaiJ9)

**Review by Fenthick — CHANGES_REQUESTED** [review]
Typo found

File: cloud_AWS/terraform/module/cloudexport.tf [hunk 7]
```diff
@@ -0,0 +1,27 @@
+terraform {
+  required_providers {
+    kentik-cloudexport = {
+      version = "0.1.0"
+      source  = "kentik.com/automation/kentik-cloudexport"
+    }
+  }
+}
+
+resource "kentik-cloudexport_item" "terraform_aws_export" {
```
[the remark this answers is not in the document]
Comment (Fenthick) [reply cloud_AWS/terraform/module/cloudexport.tf]: Resolved

File: cloud_AWS/terraform/module/variables.tf (hunk 1 above)
Comment (Fenthick) [reply cloud_AWS/terraform/module/variables.tf]: Resolved

File: cloud_AWS/terraform/module/variables.tf (hunk 2 above)
Comment (Fenthick) [reply cloud_AWS/terraform/module/variables.tf]: Resolved

File: cloud_AWS/terraform/module/variables.tf (hunk 3 above)
Comment (Fenthick) [reply cloud_AWS/terraform/module/variables.tf]: Resolved

File: cloud_AWS/terraform/module/variables.tf (hunk 4 above)
Comment (Fenthick) [reply cloud_AWS/terraform/module/variables.tf]: Resolved

**Review by Fenthick — APPROVED** [review]

**Review by mateuszmidor — APPROVED** [review]

**Review by mmac-m3a — APPROVED** [review]

**Pull request merged**